pkg/upgraders/healthcheck_pdb.go

```diff
@@ -44,11 +45,31 @@ func checkPodDisruptionBudgets(c client.Client, logger logr.Logger) (string, err
 		logger.Info("unable to list PodDisruptionBudgets/v1")
 		return metrics.PDBQueryFailed, err
 	}
-
+	var value string
 	for _, pdb := range pdbList.Items {
 		if !strings.HasPrefix(pdb.Namespace, "openshift-*") || checkNamespaceExistsInArray(namespaceException, pdb.Namespace) {
-			if pdb.Spec.MaxUnavailable != nil && pdb.Spec.MaxUnavailable.IntVal == 0 {
+			if pdb.Spec.MaxUnavailable != nil {
```

Comment:
I'm OK with the current implementation. 
Just wonder if this is better
`if pdb.Spec.MaxUnavailable != nil && ((pdb.Spec.MaxUnavailable.Type == intstr.Int) && (pdb.Spec.MaxUnavailable.IntVal == 0)) `

Reply:
yeah this can be implemented. For safe side I added for string "0" as well.

Reply:
Just a discussion. Thanks for taking it. 
Your concern is valid. However, I remember user is not able to set string "0" here.